### ISSUE TITLE
Feature request: alternative way to deliver configs for custom more-info popups

### ISSUE THREAD

**Issue by kulmegil** [thomasloven/hass-browser_mod#361]
Currently the only way to replace a more-info popup is to create and add popup-card into each / specific view.

This is a serious limitation. Either each popup-card needs to be added to every view OR we have to navigate into specific view + make a delay for rendering - painful especially from a server call. 

This kind of defeats the idea of replacing the more info dialogs in the first place. This should work over the scope of entire dashboard - like it was possible in previous version. Please consider adding alternative place to define the configs instead of just each view.

**Comment by Alfiegerner** [comment]
@kulmegil does refer to the removal of the popup_cards config option? Just checking if I'm on the same page.

**Comment by kulmegil** [comment]
@Alfiegerner yes. 
The key could be put inside each view or in the root of dashboard config. The latter is the one I used exclusively.

**Comment by DarkFox** [comment]
The removal of `popup_cards` has had a larger impact, than I think the authors expected.

The new method is fine for cards that support `tap_action`, but cards like gauge and weather, don't.
So now there's no way to replace the more-info popup, for those cards.

**Comment by parrel** [comment]
I also preferred the old way. The cards I created, still take up a bit of space on my dashboard, even when hidden.

**Comment by Dilergore** [comment]
I've been using the popup_cards section for a very long time. It was a very easy and clean way to define the popups. I really hope this will be added very soon to the new version.

**Comment by ASNNetworks** [comment]
Yes, this change is the reason I still don't want to update to 2.0. It's completely defeats the purpose of replacing more-info dialogs globally.

**Comment by dcapslock** [comment]
Noting there is a new browser_mod Version 2.3.4 available via my fork. See [V2.3.4 Disussion](https://github.com/dcapslock/hass-browser_mod/discussions/17). This version looks for popup card that matches the entity in the current dashboard (not just view). Looking at the description of the issue, it will likely be what you are after (tested just now).

**Comment by dcapslock** [comment]
NOTE: PR #853 closed in favour for UI method.

**Comment by dcapslock** [comment]
[v2.4.0 Released](https://github.com/thomasloven/hass-browser_mod/releases/tag/v2.4.0). This allows a custom:popup-card to be used in all views of a Dashboard.
